core/src/main/scala/tech/beshu/ror/accesscontrol/factory/RawRorConfigBasedCoreFactory.scala

```diff
@@ -317,7 +319,11 @@ class RawRorConfigBasedCoreFactory(esVersion: EsVersion)
             dynamicVariableTransformationAliases.items.map(_.alias)
           )
         )
-        auditingTools <- AsyncDecoderCreator.from(AuditingSettingsDecoder.instance(esVersion))
+        auditEnvironmentContext = new AuditEnvironmentContext {
```

Comment:
let's create a non-anonymous class for it (like we did with `AuditRequestContextBasedOnAclResult`)

Reply:
done